systemd/reload.sls

```diff
@@ -0,0 +1,4 @@
+reload_systemd_configuration:
+  cmd.wait:
+    - name: systemctl daemon-reload
+    - runas: root
```

Comment:
Presumably `root` is default `runas` user but salt api docs do not confirm this. So yeah, this is safer.